.gitignore

```diff
@@ -3,7 +3,6 @@
 
 # Packages
 vendor/
-composer.lock
```

Comment:
I think that decision was to not include composer.lock in libraries, can you confirm that @healerz?

Reply:
I've changed it back.